cfg/binds.cfg

```diff
@@ -87,7 +88,7 @@ bind 	F8 	"record fix; stop; snd_restart; hud_reloadscheme" //This will freese y
 bind	F9	"load_itempreset 0; wait 10; load_itempreset 2; wait 10; load_itempreset 0"
 bind	F10	"load_itempreset 1; wait 10; load_itempreset 2; wait 10; load_itempreset 1"
 bind 	F11	"load_itempreset 2; wait 10; load_itempreset 3; wait 10; load_itempreset 2"
-bind	F12	"load_itempreset 3; wait 10; load_itempreset 2; wait 10; load_itempreset 3"
+bind	F12	 show_quest_log 

```

Comment:
This key is taken. Perhaps choose another one?

Reply:
you are right, sorry about that.
I've just updated it and bind the show_quest_log to p